### PR TITLE
Add Agones Python SDK; make the web server an Agones GameServer; update skaffold and README

### PR DESCRIPTION
**Pull request by gongmax**
The src/agones is just the generated Python SDK. Feel free to ignore them when reviewing.

### CODE REVIEW

File: examples/guess-the-sketch/src/app.py [hunk 1]
```diff
@@ -34,8 +42,36 @@
...
+    except:
+        time.sleep(2)
+        logger.info('retry connection')
+
+def agones_health():
+    while True:
+        try:
+            agones.health(body)
+            time.sleep(0.3)
+            logger.info('health check passed')
```
Comment (roberthbailey) [inline examples/guess-the-sketch/src/app.py:67]: This line is being printed in the logs very, very often. I think you could slow down the health interval a bit (maybe every second or two)?

**Comment by roberthbailey** [comment]
Also, there are a few conflicts that need to be resolved before this can be merged.

File: examples/guess-the-sketch/src/app.py (hunk 1 above)
Comment (gongmax) [reply examples/guess-the-sketch/src/app.py]: I have the impression that large interval for health check didn't work for Python client: https://github.com/googleforgames/agones/issues/2401. But I increased it to 2 and seems works, so updated it.

**Review by roberthbailey — APPROVED** [review]

File: examples/guess-the-sketch/README.md [hunk 2]
```diff
@@ -12,19 +12,17 @@ The gameplay is:
...
-`http://genai-api.genai.svc/genai/image` endpoints need to be running.
-
+1. Follow the top-level README (ensure that the http://genai-api.genai.svc/genai/text 
+   and http://genai-api.genai.svc/genai/image endpoints are running).
+2. [Install Agones](https://agones.dev/site/docs/installation/install-agones/yaml/#installing-agones) in your cluster.
+3. Install Guess the Sketch into your cluster:
 ```
 cd ~/GenAI-quickstart/examples/guess-the-sketch
-skaffold run --build-concurrency=0
+skaffold run --build-concurrency=0 --cache-artifacts=false
```
Comment (zmerlynn) [inline examples/guess-the-sketch/README.md:21]: You really only need this if you're iterating on it, right?

**Review by zmerlynn — APPROVED** [review]

**Pull request merged**